ai/WFLY-19381-[EXPERIMENTAL]-Provide_a_galleon_feature_pack_to_facilitate_Genrative_AI_development.adoc

```diff
@@ -0,0 +1,264 @@
+---
+categories:
+  - ai
+# - core
+# - management
+# if missing, add it to _data/widfly-categories and use its id
+---
+= [Experimental] Provide a Galleon feature pack to facilitate Generative AI application development
+
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The goal of this feature is to provide a simple way to develop Generative AI applications. 
+This could be done by allowing those resources to be configured in WildFly and then injected into the application via CDI to be used there.
+Given that currently one of the main use-case of Generative AI is to provide Retrieval-Augmented Generation application, all the elements required for such an application should be accessible.
+The feature should take inspiration using LangChain4J which provides an API over several LLM inferers to build such applications.
+As an experimental feature, we will provide LangChain4J integration instead of providing our own API, defining resources to be injected into applications in an *ai* subsystem.
```

Comment:
I think that we always use the "AI" in upper case, instead of "ai" in lower case, since it's an acronym.

Reply:
This is the name of the subsystem thus the lower case